src/khoj/interface/web/content_type_github_input.html

```diff
@@ -35,9 +35,9 @@ <h4>Repositories</h4>
                 {% endfor %}
             </div>
             <button type="button" id="add-repository-button">Add Repository</button>
-            <h4>You probably don't need to edit these.</h4>
+            <h4 style="display: none;">You probably don't need to edit these.</h4>
```

Comment:
Should we just delete this line? Like you've done in the `processor_conversation_input.html`